web/tests/i18n.test.js

```diff
@@ -41,10 +41,10 @@ run_test("$t", () => {
     assert.equal(
         $t(
             {
-                id: "You subscribed to stream {stream}",
-                defaultMessage: "You subscribed to stream {stream}",
+                id: "You subscribed to channel {name}",
+                defaultMessage: "You subscribed to channel {name}",
```

Comment:
As of commit 3e6f34780a28, this is no longer an example of `$t` in the codebase. I don't know if it particularly matters for these tests, so I just updated it to use the modern convention of channel.

Reply:
Yeah, I don't think it matters.